src/sql/workbench/common/views.ts

```diff
@@ -29,7 +29,8 @@ export interface ITreeItem extends vsITreeItem {
 	payload?: IConnectionProfile; // its possible we will want this to be more generic
 	sqlIcon?: string;
 	type?: NodeType;
-	nodeInfo?: NodeInfo
+	nodeInfo?: NodeInfo;
+	sessionId?: string;
```

Comment:
Add doc comment for what a "session" is in this context

Reply:
done